lib/ddtrace/contrib/rails/active_support.rb

```diff
@@ -22,21 +43,68 @@ def self.instrument
           end
         end
 
-        def self.trace_cache(resource, _name, start, finish, _id, payload)
-          tracer = ::Rails.configuration.datadog_trace.fetch(:tracer)
+        def self.create_span(tracer)
           service = ::Rails.configuration.datadog_trace.fetch(:default_cache_service)
           type = Datadog::Ext::CACHE::TYPE
+          tracer.trace('rails.cache', service: service, span_type: type)
```

Comment:
yes in this case it's ok to bind Redis to ``rails.cache`` since we're in a Cache handler